### PR TITLE
public webroot based installation, this keeps the vendor directory non-visible

### PR DESCRIPTION
**Pull request by KorvinSzanto**
# Todo
* [x]  Switch to upstream when https://github.com/composer/installers/pull/339 merges
* [ ] Better instructions for managing installations over time

### CODE REVIEW

**Review by olsgreen — COMMENTED** [review]
@KorvinSzanto What are your thoughts on making the cli install run at the end of the composer install routine? Reduces the install to one command?

Also the install instructions here are wrong I'll submit a corrected version.

**Comment by KorvinSzanto** [comment]
@olsgreen I think that should be a separate command honestly since you don't always want to install straight away

File: README.md [hunk 1]
```diff
@@ -7,10 +7,7 @@ $ cd newproject
 # Use composer to prepare the core
 $ composer install
 
-# Use the concrete5 CLI tool to run the c5 install
-$ ./vendor/bin/concrete5 c5:install -i
+# Use the concrete5 CLI tool to run the c5 install from the public directory
+$ cd public
+$ ./concrete/bin/concrete5 c5:install -i
```
Comment (fabian) [inline README.md:12]: This should stay `./vendor/bin/concrete5 c5:install -i`, right?

Comment (KorvinSzanto) [reply README.md:12]: I believe this needs to run in the public directory unfortunately. ./concrete/bin/concrete5 c5:install works okay

**Review by fabian — APPROVED** [review]

**Review by Mnkras — APPROVED** [review]

**Pull request merged**